channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1079,7 +1051,7 @@ static UINT rdpdr_process_connect(rdpdrPlugin* rdpdr)
 		{
 			RDPDR_DRIVE* drive = (RDPDR_DRIVE*)device;
 
-			if (drive->Path && (strcmp(drive->Path, "*") == 0))
+			if (!rdpdr->hotplugThread && drive->Path && (strcmp(drive->Path, "*") == 0))
```

Comment:
Might be good, if you allow more than one wildcard.
Still, adding wildcard devices (see below) does not seem to make sense.

Reply:
That was some defensive programming. 
If only one wild card could ever happen, I can remove the check.